CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+## 0.51.0-rc.0 / 2025-05-12
+
+We are finally reaching the release candidate for the 0.51.0 version. 
+We are reaching a confident level of stability, and we should not expect any major changes before the final release.
+
+In this new release, we are introducing new features and enhancements, including:
+
+- A new plugin list page in the UI, allowing users to view and manage installed plugins.
+- The ability to configure CORS settings for the application.
+- Improvements to the CLI, including a command to list installed plugins and enhancements to the plugin development process.
+- The addition of zoom in/zoom out buttons.
+- Support for Datasource Variables, enabling users to swap the datasource using the variable.
```

Comment:
I wouldnt mention this one since without https://github.com/perses/plugins/pull/110 it brings no added value to end users yet

Reply:
removed